Dockerfile

```diff
@@ -0,0 +1,61 @@
+# Building this image will include the running of all benchmarks in a "cold
+# cache" state, and thus will take quite a bit of time (roughly 30-60 minutes,
+# with Ganache excluded).  The cache will be retained in the built image, such
+# that running the image will execute benchmarks with a "warm cache". For
+# example:
+#
+#   docker --build-arg ALCHEMY_API_KEY=... -t sim .
```

Comment:
I'm a docker amateur so may be doing something wrong, but running this command gives me the error below.

```
$ docker --build-arg ALCHEMY_API_KEY=<myKey> -t sim .
unknown flag: --build-arg
See 'docker --help'.
```

Using macOS 11.6.2 and `docker version` gives this output: https://app.warp.dev/block/4gBXpbdr0X86tjbkRElxqL

Reply:
Oops! My bad! I simply made a manual error in that command. Addressed in https://github.com/mds1/convex-shutdown-simulation/pull/5/commits/f572e96fcf22aea65e41fdc111df78666d5dbec9

Sure would be nice if there were a GitHub Action or other CI thing that would exercise this thing and provide an authoritative command source :)

Reply:
Awesome, this is great + working as expected now.

Good call on CI, just created https://github.com/mds1/convex-shutdown-simulation/issues/6 to track